app/templates/src/handlebars/helpers/helpers.js

```diff
@@ -1,5 +1,5 @@
 /**
```

Comment:
This file doesn’t see to be the latest version. In terms of filename and content.
See https://github.com/micromata/bootstrap-kickstart/blob/v4-dev/src/handlebars/helpers/addYear.js